docs/docs/workflows/workspaces/README.md

```diff
@@ -0,0 +1,63 @@
+# Workspaces
+
+When registering a Pipedream account, you're prompted to create a workspace. Workflows are organized and contained within your workspace.
+
+Workspaces are a container for your projects and workflows, and can be shared with your team members.
+
+<div class="flex justify-center">
+  <img src="https://res.cloudinary.com/pipedreamin/image/upload/v1674062859/docs/CleanShot_2023-01-18_at_12.27.13_itqa7f.png" alt="How workspaces are organized" />
+</div>
+
+## Projects
+
+A workspaces can contain one or more _projects_. Projects are a way to organize your workflows into specific groupings or categories.
+
+::: details How limit workspace members from viewing or developing workflows in certain projects?
+
+Members of a given workspace can see all projects in the workspace. 
+
+If you need to limit certain project's visibility, to only some members, then those members belong in a different workspace.
+
+:::
+
+### Creating Projects
+
+To create a new project, first [open the Projects section in the dashboard](https://pipedream.com/projects).
+
+Then click **Create project** to start a new project.
+
+Enter in your desired name for the project in the prompt, then click **Create**.
+
+That's it, you now have a dedicated new project created within your workspace. Now you can create workflows within this project, or move workflows into it, or create folders for further organization.
+
+### Creating folders and workflows in projects
+
+Within a given project, you can create folders for your workflows.
+
+Open your project, and then click the **Create** button for a dropdown to create a workflow in your current project.
+
+::: tip
+
+Helpful hotkeys to speed up your development
```

Comment:
Bottom margin looks a little off at the bottom of this `tip` section. I think it happens with all lists in our Vuepress setup (no margin below list elements)

<img width="715" alt="Screen Shot 2023-01-18 at 12 19 37 PM" src="https://user-images.githubusercontent.com/242668/213285962-a558e64b-11e3-4621-b8d0-dc72f519d4d5.png">